.spi.yml

```diff
@@ -21,3 +21,5 @@ builder:
       - SwiftSyntaxMacros
       - SwiftSyntaxMacroExpansion
       - SwiftSyntaxMacrosTestSupport
+      custom_documentation_parameters: [--experimental-skip-synthesized-symbols]
+      swift_version: '5.9'
```

Comment:
Ethan mentioned the flag should be supported on 5.8 as well, so this additional key is perhaps not necessary. I can run a test before merging.

Reply:
I've confirmed this works on 5.8 and with the double dash command.

```
Size (/MB): 146.0
File count: 16557
```